<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorginForm</class>
 <widget class="QWidget" name="WorginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Worgin</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/worginicon.svg</normaloff>:/icons/worginicon.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Address of The Target:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addressEdit">
         <property name="text">
          <string>bbc.com</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Words of Interest:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="wordsList">
         <item>
          <property name="text">
           <string>coronavirus</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QToolButton" name="removeBtn">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/minus.svg</normaloff>:/icons/minus.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>New Word:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="newWordEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="addBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/plus.svg</normaloff>:/icons/plus.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Logs:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startBtn">
           <property name="text">
            <string>Worgin</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/worginicon.svg</normaloff>:/icons/worginicon.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="logOutput"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addBtn</sender>
   <signal>clicked()</signal>
   <receiver>WorginForm</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startBtn</sender>
   <signal>clicked()</signal>
   <receiver>WorginForm</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeBtn</sender>
   <signal>clicked()</signal>
   <receiver>WorginForm</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
